test/fp-intrinsics.ll

```diff
@@ -0,0 +1,346 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Could you please move this file into `test/llvm-intrinsics` directory?

Reply:
Done.